pandas/core/window/rolling.py

```diff
@@ -658,9 +658,22 @@ def _numba_apply(
             return self._resolve_output(out, obj)
 
     def aggregate(self, func, *args, **kwargs):
-        result = ResamplerWindowApply(self, func, args=args, kwargs=kwargs).agg()
+        # GH46132
```

Comment:
Could you write a comment (or link to an existing issue) that this should not be needed once ResamplerWindowApply supports axis=1?

Reply:
Sure.

Reply:
Sorry, I meant a different existing issue if one exists

Reply:
Ok, I will update the comment.